platform/jvm/capture/src/test/kotlin/io/bitdrift/capture/FatalIssueReporterProcessorTest.kt

```diff
@@ -73,9 +117,45 @@ class FatalIssueReporterProcessorTest {
             eq(FatalIssueType.ANR),
             fatalIssueReportCaptor.capture(),
         )
-        assertThat(fatalIssueReportCaptor.firstValue.type).isEqualTo(FatalIssueType.ANR)
-        assertThat(fatalIssueReportCaptor.firstValue.errors[0].reason).isEqualTo("ANR reason wip")
-        assertThat(fatalIssueReportCaptor.firstValue.errors[0].name).isEqualTo(description)
+        assertThat(fatalIssueReportCaptor.firstValue.errors[0].reason)
+            .isEqualTo(APP_EXIT_DESCRIPTION_ANR)
+        assertThat(fatalIssueReportCaptor.firstValue.errors[0].name)
+            .isEqualTo("at io.bitdrift.capture.FatalIssueGenerator.startProcessing(FatalIssueGenerator.kt:106)")
+        assertThat(
+            fatalIssueReportCaptor.firstValue.errors[0]
+                .stackTrace[0]
+                .type,
+        ).isEqualTo(1)
+        assertThat(
+            fatalIssueReportCaptor.firstValue.errors[0]
+                .stackTrace[0]
+                .state,
+        ).isEmpty()
+        assertThat(
+            fatalIssueReportCaptor.firstValue.errors[0]
+                .stackTrace[0]
+                .className,
+        ).isEqualTo("io.bitdrift.capture.FatalIssueGenerator")
+        assertThat(
+            fatalIssueReportCaptor.firstValue.errors[0]
+                .stackTrace[0]
+                .symbolName,
+        ).isEqualTo("io.bitdrift.capture.FatalIssueGenerator.startProcessing")
+        assertThat(
+            fatalIssueReportCaptor.firstValue.errors[0]
+                .stackTrace[0]
+                .sourceFile.path,
+        ).isEqualTo("FatalIssueGenerator.kt")
+        assertThat(
+            fatalIssueReportCaptor.firstValue.errors[0]
+                .stackTrace[0]
+                .sourceFile.lineNumber,
+        ).isEqualTo(106)
+        assertThat(
+            fatalIssueReportCaptor.firstValue.errors[0]
+                .stackTrace[0]
+                .sourceFile.column,
+        ).isEqualTo(0)
```

Comment:
are we validating somewhere there proper parsing of `java.lang.reflect.Method.invoke`?